Editor/Window/RegisterComputeInput.cs

```diff
@@ -72,11 +79,16 @@ private async void OnRegisterComputeButtonClicked()
         {
             if (_computeState is ComputeStatus.NotRegistered or ComputeStatus.Registering)
             {
-                var success = await _computeManager.RegisterFleetCompute(_computeName, _fleetDetails.FleetId, _location, _ipAddress);
-                if (success)
+                var registerFleetCompute = await _computeManager.RegisterFleetCompute(_computeName, _fleetDetails.FleetId, _location, _ipAddress);
+                if (registerFleetCompute.Success)
                 {
                     _computeState = ComputeStatus.Registered;
                 }
+                else
+                {
+                    var url = string.Format(Urls.AwsAnywhereFleetLogs, _stateManager.Region);
+                    _registerComputeStatusBox.Show(StatusBox.StatusBoxType.Error, ErrorText + registerFleetCompute.ErrorMessage, url, URLText);
```

Comment:
The strings being used should come from the Strings factory (which are enums/constants), and not be hardcoded strings.

Reply:
Fixed